test/suite/test_rollback_to_stable03.py

```diff
@@ -115,15 +116,19 @@ def test_rollback_to_stable(self):
         self.check(valueb, uri, nrows, None, 20)
         self.check(valuea, uri, nrows, None, 10)
 
-        stat_cursor = self.session.open_cursor('statistics:', None, None)
-        calls = stat_cursor[stat.conn.txn_rts][2]
-        hs_removed = stat_cursor[stat.conn.txn_rts_hs_removed][2]
-        keys_removed = stat_cursor[stat.conn.txn_rts_keys_removed][2]
-        keys_restored = stat_cursor[stat.conn.txn_rts_keys_restored][2]
-        pages_visited = stat_cursor[stat.conn.txn_rts_pages_visited][2]
-        upd_aborted = stat_cursor[stat.conn.txn_rts_upd_aborted][2]
-        stat_cursor.close()
-
+        with WiredTigerStat(self.session) as stat_cursor:
+            calls = stat_cursor[stat.conn.txn_rts][2]
+            hs_removed = stat_cursor[stat.conn.txn_rts_hs_removed][2]
+            keys_removed = stat_cursor[stat.conn.txn_rts_keys_removed][2]
+            keys_restored = stat_cursor[stat.conn.txn_rts_keys_restored][2]
+            pages_visited = stat_cursor[stat.conn.txn_rts_pages_visited][2]
+            upd_aborted = stat_cursor[stat.conn.txn_rts_upd_aborted][2]
+            rts_btrees_applied = stat_cursor[stat.conn.txn_rts_btrees_applied][2]
+            rts_btrees_skipped = stat_cursor[stat.conn.txn_rts_btrees_skipped][2]
+
+        self.assertEqual(rts_btrees_applied + rts_btrees_skipped, 1)
```

Comment:
Should we just check for this given the assert below?
```suggestion
        self.assertEqual(rts_btrees_skipped, 0)
```

Reply:
I think we can keep check both, because still one more scenario that the `rts` call is failed earlier some reason.